### PR TITLE
Add visual question-answer feature in clevr dataset

### LINKED ISSUE #827: CLEVR dataset is missing the questions
**Short description**
CLEVR is a Visual Question Answering dataset but the questions are missing, so user needs to jump through a bunch of hoops to download and parse them

**Environment information**
* Operating System: linux
* Python version: 3.6.8
* `tensorflow-datasets`/`tfds-nightly` version: 1.1
* `tensorflow`/`tensorflow-gpu`/`tf-nightly`/`tf-nightly-gpu` version: gpu 2b1

**Reproduction instructions**

```
ds_test, ds_train, ds_validation = tfds.load(name="clevr", split=['test', 'train', 'validation'])
for element in ds_train:
    tf.print(element)
```

**Link to logs**
```
 [[115 114 114]
  [117 116 114]
  [115 114 113]
  ...
  [156 153 147]
  [156 153 147]
  [156 153 148]]],
 'objects': {'3d_coords': [[0.438755214 -2.79463482 0.7]
 [2.59563708 1.5761956 0.35]
 [-2.9369 2.04038 0.35]
 ...
 [2.28646755 2.97107983 0.7]
 [-1.16437554 2.7382021 0.7]
 [2.27009153 0.293746442 0.7]],
             'color': [1 6 0 ... 3 2 2],
             'material': [0 0 1 ... 0 0 0],
             'pixel_coords': [[141 187 9.35730934]
 [383 166 10.7077045]
```

**Expected behavior**
i expected a visual question answering dataset to have questions to answer

**Additional context**

### PR DESCRIPTION
**Pull request by vijayphoenix**
Fix #827

`data_info.json`: [link](https://gist.github.com/vijayphoenix/ededc2f2ea060b5bc7d918944d2b216f)
This gist only contains info for validation set because of some computational limits.

### CODE REVIEW

**Comment by vijayphoenix** [comment]
@ChanchalKumarMaji @cyfra, please review.
Thank you

File: tensorflow_datasets/image/clevr.py [hunk 1]
```diff
@@ -56,6 +56,10 @@ def _info(self):
         features=tfds.features.FeaturesDict({
             "image": tfds.features.Image(),
             "file_name": tfds.features.Text(),
+            'question_answer': tfds.features.Sequence({
```
Comment (ChanchalKumarMaji) [inline tensorflow_datasets/image/clevr.py]: Just use `""` as other keys to look good. Thanks !

**Comment by ChanchalKumarMaji** [comment]
The `fake_samples` seems to be taken from the original dataset. Can u please change that.

**Comment by vijayphoenix** [comment]
Edited.

**Comment by vijayphoenix** [comment]
I think this PR should not be labeled as a `dataset request`.
